packages/core/src/rules/spot/parameters-no-body-inside-in.ts

```diff
@@ -0,0 +1,17 @@
+import type { ArazzoRule } from '../../visitors';
+import type { UserContext } from '../../walk';
+
+export const ParametersNoBodyInsideIn: ArazzoRule = () => {
+  return {
+    Parameter: {
+      enter(parameter, { report, location }: UserContext) {
+        if (parameter.in === 'body') {
+          report({
+            message: 'Parameters `in` property not allowed to have `body` value.',
```

Comment:
Technically it is allowed, but we don't support that. What about this:

```suggestion
            message: 'The `body` value of the `in` property is not supported.',
```

Reply:
'The `body` value of the `in` property is not supported by Spot'